### ISSUE TITLE
False positive: LazilyInitializedFields

### ISSUE THREAD

**Issue by jakobnissen** [julia-vscode/StaticLint.jl#369]
MWE:
```julia
module Foo

using LazilyInitializedFields: @lazy

@lazy mutable struct Bar
    @lazy x::Int
end

end
```
This requires `LazilyInitializedFields` to be in the project.

VSCode's linter reports:
`Missing reference: x`
